tests/nightly/gpu/test_rag.py

```diff
@@ -388,5 +398,70 @@ def test_bart_fid_rag_dpr_poly(self):
         _test_zoo_file(FID_RAG_DPR_POLY_ZOO_MODEL, True, True)
 
 
+class TestLoadDPRModel(unittest.TestCase):
+    """
+    Test loading different DPR models for RAG.
+
+    See RagAgent._should_override_dpr_model_weights for important note
+    regarding specifying the *same* dpr model file as was used to train
+    the model.
+    """
+
+    def test_load_dpr(self):
+        opt = ParlaiParser(True, True).parse_args([])
+        # First, we'll load up a DPR model from the zoo dpr file.
+        default_query_encoder = DprQueryEncoder(
+            opt, dpr_model='bert', pretrained_path=DPR_ZOO_MODEL
+        )
+        query_encoder = DprQueryEncoder(
+            opt,
+            dpr_model='bert_from_parlai_rag',
+            pretrained_path=RAG_SEQUENCE_ZOO_MODEL,
+        )
+        assert not torch.allclose(
```

Comment:
nit: why not using the `unittest` functions like `assertIsNone` and `assertTrue` etc.?

Reply:
i've spoken with @stephenroller about this and he's told me that it's all the same to pytest